lib/Config.php

```diff
@@ -662,4 +663,18 @@ public function getGridVideosLimit(): int {
 	public function getGridVideosLimitEnforced(): bool {
 		return $this->config->getAppValue('spreed', 'grid_videos_limit_enforced', 'no') === 'yes';
 	}
+
+	/**
+	 * User setting falling back to admin defined app config
+	 *
+	 * @param ?string $userId
+	 * @return bool
+	 */
+	public function getCallsStartWithoutMedia(?string $userId): bool {
+		if ($userId !== null && $this->config->getUserValue($userId, 'spreed', UserPreference::CALLS_START_WITHOUT_MEDIA) === 'no') {
```

Comment:
That's supposed to evaluate to `true` here?
![image](https://github.com/user-attachments/assets/c386f795-0cad-43c5-9569-c0fdb4cdbc6a)
<img width="128" alt="image" src="https://github.com/user-attachments/assets/0cb6271f-12f5-4454-9735-6c474fbf1b5c">

Reply:
Ops, only `no` was respected. fixed now